packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/components/AddWalletButton/index.tsx

```diff
@@ -65,11 +65,30 @@ const AddWalletButton = ({ device, instances, addDeviceInstance, selectDeviceIns
         });
     };
 
+    type IsLockedDeviceWrapperProps = {
+        children: React.ReactElement;
+        isLocked: boolean;
+        wrapper: (children: React.ReactElement) => JSX.Element;
+    };
+
+    const IsLockedDeviceWrapper: React.FC<IsLockedDeviceWrapperProps> = ({
+        isLocked,
+        wrapper,
+        children,
+    }) => (isLocked ? wrapper(children) : children);
+
     return (
         <AddWallet>
-            <StyledTooltip
-                content={<Translation id="TR_TO_ACCESS_OTHER_WALLETS" />}
-                cursor="pointer"
+            <IsLockedDeviceWrapper
+                isLocked={isLocked}
```

Comment:
Nitpick: When you click on the button, the device become locked so the tooltip appears for a moment.